internal/datacoord/compaction_l0_view.go

```diff
@@ -73,39 +75,49 @@ func (v *LevelZeroSegmentsView) Equal(others []*SegmentView) bool {
 	return diffCount == 0
 }
 
-// ForceTrigger triggers all qualified LevelZeroSegments according to views
-func (v *LevelZeroSegmentsView) ForceTrigger() (CompactionView, string) {
-	// Only choose segments with position less than the earliest growing segment position
-	validSegments := lo.Filter(v.segments, func(view *SegmentView, _ int) bool {
-		return view.dmlPos.GetTimestamp() < v.earliestGrowingSegmentPos.GetTimestamp()
-	})
-
-	targetViews, reason := v.forceTrigger(validSegments)
-	if len(targetViews) > 0 {
-		return &LevelZeroSegmentsView{
-			label:                     v.label,
-			segments:                  targetViews,
-			earliestGrowingSegmentPos: v.earliestGrowingSegmentPos,
-		}, reason
-	}
-
-	return nil, ""
-}
-
```

Comment:
Please add back ForthTrigger. It'll be used for other reasons like: System IDLE, Scheduler IDLE,  configured IDLE time duration, or MixCompaction not meeting minimum compaction condition, etc.

Although it seems not useful, it's Not a l0 compaction only thing.

Reply:
OK